### PR TITLE
OBSCDOCS-1374 - Loki Statement of Purpose addition to Logging 6.0 docs. 

### PR DESCRIPTION
**Pull request by libander**
Version(s):  4.14 , 4.15, 4.16, 4.17

Issue: [OBSDOCS-1374](https://issues.redhat.com//browse/OBSDOCS-1374) & [OBSDOCS-1425](https://issues.redhat.com//browse/OBSDOCS-1425)

Link to docs preview: https://83592--ocpdocs-pr.netlify.app/openshift-enterprise/latest/observability/logging/logging-6.0/log6x-loki.html

QE review:
- [x] QE has approved this change.

### CODE REVIEW

**Comment by libander** [comment]
QE review: @anpingli 
SME review: @alanconway / @cahartma / @jcantrill

**Comment by anpingli** [comment]
LGTM

**Review by JoaoBraveCoding — APPROVED** [review]
/lgtm

**Review by jneczypor — COMMENTED** [review]
/remove-label peer-review-in-progress
/remove-label peer-review-needed
/label peer-review-done

**Comment by libander** [comment]
@anpingli - re-review, please. 
@r2d2rnd - does this work?

**Comment by libander** [comment]
@anpingli / @r2d2rnd - Re-review, please.

File: observability/logging/logging-6.0/log6x-loki.adoc [hunk 1]
```diff
@@ -17,13 +19,13 @@ You can configure a `LokiStack` CR to store application, audit, and infrastructu
 
 === Core Setup and Configuration
 *Role-based access controls, basic monitoring, and pod placement to deploy Loki.*
-
+====
```
Comment (kcarmichael08) [inline observability/logging/logging-6.0/log6x-loki.adoc:22]: I don't think this should be here - I think this is why there are erroneous `=` and the headers aren't showing properly for the sections following this one in the preview (see screenshot). 
Screenshot 2024-10-21 at 9 02 38 AM

**Review by kcarmichael08 — COMMENTED** [review]
There are some formatting issues with headings, see attached screenshot. Also, I know this isn't changed text, but I don't think the bolded sentence fragments after each heading and the procedures being contained in boxes are the way that we normally format these (see screenshot). Maybe the boxes are because of the erroneous `====`? 
Screenshot 2024-10-21 at 9 12 46 AM

**Comment by kcarmichael08** [comment]
Hi @libander , this looks good - should it also be cherry picked to 4.18 (I'm assuming yes?)

**Comment by kcarmichael08** [comment]
Hi @libander , I still had this window open - it doesn't look like the manual cherry pick to enterprise-4.14 was done. Just didn't want it to get lost.

**Pull request merged**